Dockerfile

```
@@ -0,0 +1,16 @@
+WORKDIR /app
+COPY gradlew gradlew
+COPY gradle gradle
+COPY build.gradle settings.gradle ./
+RUN chmod +x gradlew
+RUN ./gradlew --no-daemon dependencies || true
+COPY src src
+RUN ./gradlew --no-daemon bootJar
+
+FROM eclipse-temurin:17-jre
+WORKDIR /app
+RUN apt-get update && apt-get install -y --no-install-recommends curl && rm -rf /var/lib/apt/lists/*
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)